en/assembly/quick_start_cuav_v5_nano.md

```diff
@@ -14,8 +14,8 @@ We'll go through each of these in detail in the following sections.
 
 | Main interface | Function |
 | :--- | :--- |
-| POWER1 | Connect Power module;  power input & AD voltage  and current detection. |
-| POWER2 | Connect i2c smart battery. |
+| Power | Connect Power module;  power input & AD voltage  and current detection. |
+| PM2 | Connect i2c smart battery. |
```

Comment:
@cuhome - I changed this to `PM2` - that is the label on the device right?

Is there a particular smart battery you are recommending? Is there any advice for this power connection?

Reply:
No, can't be modified like this, power2 is the i2c interface, it is not an adc interface.

Reply:
It is power1 and power2 on the silkscreen, so the modification may not know which power interface is.

Reply:
@cuhome I'm confused. This is for the nano doc, which does not have power 1 and power 2 as silkscreen, but instead `Power` and (I think) `PM2` on the side?

Reply:
@cuhome Not as far as I can see on the Nano. Are you sure? Point is, the label should match what they see on the device.